src/mui/field/PhoneField.js

```diff
@@ -17,7 +17,7 @@ export const localPhoneFormat = (parsed_number, locale) => {
     return parsed_number;
 };
 
-const PhoneField = ({ source, record, elStyle, locale, linkType }) => {
+const PhoneField = ({ source, record, elStyle, locale, linkType, text }) => {
```

Comment:
I'm not a big the props name.
Moreover it only work if linkType is set.
I would suggest linkLabel and to change the if line 33 to if (linkType || linkLabel)

Reply:
Ok for the wording, but I don't get your point for line 33.